extensions/wikia/CommunityPage/models/CommunityPageSpecialCommunityPolicyModel.class.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+class CommunityPageSpecialCommunityPolicyModel {
+	public function getData() {
+		global $wgUser;
+
+		return [
+			'policyUrl' => $this->getPolicyLink(),
+			'editPolicyUrl' => $this->getPolicyTextEditLink(),
+			'policyHeading' => wfMessage( 'communitypage-policy-heading' )->plain(),
+			'policyText' => wfMessage( 'communitypage-policy-text' )->plain(),
+			'policyEdit' => wfMessage( 'communitypage-policy-edit' )->plain(),
+			'policyView' => wfMessage( 'communitypage-policy-view' )->plain(),
+			'showEditLink' => in_array( 'sysop', $wgUser->getEffectiveGroups() ),
+		];
+	}
+
+	private function getPolicyLink() {
+		$title = Title::newFromText(
+			wfMessage( 'communitypage-policy-pagetitle' )->inContentLanguage()->plain(),
+			NS_HELP
+		);
+
+		if ( $title instanceof Title ) {
+			return $title->getFullURL();
+		}
+
+		return '';
+	}
+
+	private function getPolicyTextEditLink() {
+		$title = Title::newFromText(
+			'communitypage-policy-text',
```

Comment:
I think you meant communitypage-policy-pagetitle
If we want to restrict it to MW ns please write so in qqq so translators would know that we don't want article ns prefix as we add it ourselves.